package.json

```diff
@@ -1,24 +1,26 @@
 {
-  "name"        : "madoko",
-  "author"      : "Daan Leijen, Microsoft Corp.",
-  "version"     : "1.1.6",
-  "homepage"    : "http://madoko.codeplex.com",
-  "description" : "Madoko is a fast scholarly Markdown processor written in Koka",
-  "licenses": [{
-    "type"      : "Apache License 2.0", 
-    "url"       : "http://www.apache.org/licenses/LICENSE-2.0.html"
-  }],
+  "name": "madoko",
+  "author": "Daan Leijen, Microsoft Corp.",
+  "version": "1.1.6",
+  "homepage": "http://madoko.codeplex.com",
+  "description": "Madoko is a fast scholarly Markdown processor written in Koka",
+  "licenses": [
+    {
+      "type": "Apache License 2.0",
+      "url": "http://www.apache.org/licenses/LICENSE-2.0.html"
+    }
+  ],
   "keywords": [
-      "Madoko",
-      "Markdown",
-      "LaTeX",
-      "Koka", 
-      "Microsoft",
-      "Scholarly",
-      "Academic"
+    "Madoko",
+    "Markdown",
+    "LaTeX",
+    "Koka",
+    "Microsoft",
+    "Scholarly",
+    "Academic"
   ],
   "preferGlobal": true,
-  "main": "./lib/madoko.js",
+  "main": "./lib/api.js",
```

Comment:
Here I changed the main file to access `analyze` instead of `madoko`. We need to change this to allow both.

Reply:
I merged the two interfaces and changed it back, so it should be mostly backwards compatible , modulo the change  of the return type of `markdown`:

```diff
- (string,options)
+ (string,options,list<block>,formatContext)
```